controllers/patchReviewVotes.controller.js

```diff
@@ -0,0 +1,9 @@
+const { editReviewVotes } = require(`../models/editReviewVotes.model`);
+
+exports.patchReviewVotes = (req, res, next) => {
+  const reviewId = req.params.review_id;
+  const inc_votes = req.body.inc_votes;
+  editReviewVotes(inc_votes, reviewId)
+    .then((updatedReview) => res.status(200).send({ updatedReview }))
```

Comment:
I would rename what is returned and sent back to the user to be exactly what the thing is... in this case it would just be `review` instead of updatedReview.